test/testdata/deployednettemplates/recipes/custom/README.md

```diff
@@ -1,5 +1,7 @@
 # Custom Recipe
-This custom recipe serves as a template for performance testing on algonet (new network on AWS EC2 machines). With this recipe, you can modify the number of nodes, the type of machines, introduce new parameters to modify the network's configs and consensus parameters.
+This custom recipe serves as a template for performance testing on algonet (new network on AWS EC2 machines).
```

Comment:
Not new to this PR, but the other templates don't require `algonet` (just `netgoal`, I think). I notice that algonet is not public, so this might be a source of confusion.

Reply:
All of these recipes actually are spun up using algonet (which I think uses netgoal?). When we get a chance for refactoring, these recipes should probably move off of this go-algorand repo altogether.